docs/README.md

```diff
@@ -1,5 +1,10 @@
 # docs
-Here we document important concepts and functionality formica supports.
+Here we document important concepts and functionality Inago supports.
 
 ### table of contents
-- [slice expansion](slice_expansion.md)
+
+- [What is the goal of Inago?](goal.md)
+- [What do we expect from the users?](expect.md)
+- [What are the terms we use and what do they mean?](terms.md)
+- [What is the structure in fleet we generate?](fleet_structure.md)
```

Comment:
then lets drop it for now. I just made a few notes in the issues - thats not a hard written list to document.